Ports/doxygen/patches/0001-Detect-serenity-os.patch

```diff
@@ -0,0 +1,22 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Gurkirat Singh <tbhaxor@gmail.com>
+Date: Mon, 21 Aug 2023 05:47:52 +0530
+Subject: [PATCH] Detect serenity os
```

Comment:
```suggestion
Subject: [PATCH] Detect SerenityOS
```

Reply:
Also, looks like this is more of a "Pretend that SerenityOS is Linux-like"?

Reply:
Yes, I have fixed this.